src/core/trace/flexibleTracer.h

```diff
@@ -36,6 +36,8 @@ class CFlexibleTracer final {
       Internal
     };
 
+    typedef std::variant<std::monostate, BarectfPlatformFORTE, CInternalTracer> TracerVariant;
```

Comment:
I think in modern C++ instead of typedef you use useing.

Reply:
done